exampleSite/utilities/examples/content/events/yyyy-city/conduct.md

```diff
@@ -1,6 +1,7 @@
 +++
 Title = "Conduct"
 Type = "event"
+Description = ""
```

Comment:
I was trying to keep the provided-by-default fields to the absolute minimum. If we don't have a default content we want to suggest here, is it essential to leave this in? Can we just mention it in the README?